api/syfosmregister-v1.yaml

```diff
@@ -0,0 +1,355 @@
+      parameters:
+        - $ref: '#/components/parameters/sykmeldingId'
+      responses:
+        200:
+          description: Ok
+
+  /sykmeldinger/{sykmeldingId}/status:
+    post:
+      operationId: addStatus
+      tags:
+        - Sykmelding status
+      summary: Api for å lagre status for sykmelding
+      parameters:
+        - $ref: '#/components/parameters/sykmeldingId'
+      responses:
+        201:
+          description: Created
+        401:
+          description: Unauthorized
+        403:
+          description: Forbidden
+      requestBody:
+        $ref: '#/components/requestBodies/StatusRequest'
+  /sykmeldinger/{sykmeldingId}/bekreft:
+    post:
+      operationId: bekreftSykmelding
+      tags:
+        - Sykmelding status
+      summary: API for bekrefting av sykmeldinger
+      parameters:
+        - $ref: '#/components/parameters/sykmeldingId'
+      responses:
+        201:
+          description: Created
+        401:
+          description: Unauthorized
+        403:
+          description: Forbidden
+      requestBody:
+        $ref: '#/components/requestBodies/SykmeldingBekreftEvent'
+  /sykmeldinger/{sykmeldingId}/send:
+    post:
+      operationId: sendSykmelding
+      tags:
+        - Sykmelding status
+      summary: API for sending av sykmeldinger
+      parameters:
+        - $ref: '#/components/parameters/sykmeldingId'
+      responses:
+        201:
+          description: Created
+        401:
+          description: Unauthorized
+        403:
+          description: Forbidden
+      requestBody:
+        $ref: '#/components/requestBodies/SykmeldingSendEvent'
+
+
+  /api/v1/internal/sykmeldinger:
+    get:
+      operationId: getSykmeldinger
+      tags:
+        - Sykmelding
+        - Internal
+      parameters:
+        - $ref: '#/components/parameters/fnrQueryParam'
+      summary: API for saksbehandlere og interne. Returnerer liste med sykmledinger basert på fnr
+      responses:
+        200:
+          $ref: '#/components/responses/SykmeldingListResponse'
+
+
+components:
+  requestBodies:
+    StatusRequest:
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/SykmeldingStatus'
+    SykmeldingBekreftEvent:
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/SykmeldingBekreftEvent'
+    SykmeldingSendEvent:
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/SykmeldingSendEvent'
+  responses:
+    SykmeldingListResponse:
+      description: "Returnerer liste med sykmeldinger"
+      content:
+        application/json:
+          schema:
+            $ref: '#/components/schemas/sykmeldingList'
+  parameters:
+    sykmeldingId:
+      description: Id for sykmelding
+      name: sykmeldingId
+      in: path
+      required: true
+      schema:
+        type: string
+    fnrQueryParam:
+      description: Personnr
```

Comment:
For å flisespikke så er det vel fødselsnummer vi tar inn, personnummer er vel noe annet..?

Reply:
Ja jeg endrer den :)